manuscript/090_How_to_start.md

```diff
@@ -82,39 +82,39 @@ public class SortingOperationSpecification
 
-When inventing a�name, It is important to put the main focus on what result or action is expected from an object. If you do not, you quickly find it troublesome. As an example, one of my colleagues was specifying a�class `UserId` and wrote the following name for the Statement about comparison of two identifiers:
+When devising a name, it is important to put the main focus on what result or action is expected from an object. If you do not do that, it'll quickly become troublesome. As an example, one of my colleagues was specifying a class `UserId` and wrote the following name for the Statement about the comparison of two identifiers:
 
 `EqualOperationShouldPassForTwoInstancesWithTheSameUserName()`.
 
-Note that this is not from the perspective of a�single object, but rather from the perspective of an operation that is executed on it, which means that we stopped thinking in terms of object responsibilities and started thinking in terms of operation correctness, which is farther away from our assumption that we are writing a�Specification consisting of Statements. This name should be something more like:
+Note that this is not from the perspective of a single object, but rather from the perspective of an operation that is executed on it. We stopped thinking in terms of object responsibilities and started thinking in terms of operation correctness. To reflect a Statement, this name should be something more like:
 
 `ShouldReportThatItIsEqualToAnotherIdThatHasTheSameUserName()`.
 
-In general, when I find myself having trouble with naming like this, I am suspicious of the following things possibly happening:
+When I find myself having trouble with naming like this, I suspect one of the following may be the case:
 
-1.  I am not specifying a behavior class, but rather an outcome of a method
-2.  I am specifying more than one behavior
-3.  The behavior is too complicated and hence I need to change my design (more on this later)
-4.  I am naming the behavior on too low level of abstraction, putting too many details in the name - my personal rule is that I can only come to this conclusion when all the previous points fail me.
+1.  I am not specifying a behavior of a class, but rather the outcome of a method.
+2.  I am specifying more than one behavior.
+3.  The behavior is too complicated and hence I need to change my design (more on this later).
+4.  I am naming the behavior of an abstraction that is too low-level, putting too many details in the name. I usually only come to this conclusion when all the previous points fail me.
 
-### But can't the name really get too long?
+### But can't the name become too long?
```

Comment:
The word "really" played a role here, because I previously said not to worry about the length of the name. So this heading says something like: "are there really no exceptions to the rule?". I agree that "really" may not be the best word though.